ts/screens/onboarding/OnboardingNotificationsPreferencesScreen.tsx

```diff
@@ -179,6 +192,28 @@ const OnboardingNotificationsPreferencesScreen = (props: Props) => {
           ]}
         >
           {isFirstOnboarding && <View style={styles.separator} />}
+          <PreferencesListItem
+            title={I18n.t("profile.preferences.notifications.preview.title")}
+            description={
+              <>
+                {`${I18n.t(
+                  "profile.preferences.notifications.preview.description"
+                )} `}
+                <Link style={styles.mediumText} onPress={present}>
```

Comment:
instead of using the Link component overriding the text style, we should use the `LabelSmall` component

Reply:
Done it, passing the additional accessibility role